src/util/calendar-helper.ts

```diff
@@ -1,9 +1,6 @@
 import dayjs, { Dayjs } from "dayjs";
 
 export namespace CalendarHelper {
-    // =============================================================================
-    // HELPER FUNCTIONS
-    // =============================================================================
```

Comment:
might have miscommunicated - this header was meant to be moved down, above the unexported functions